gdcdictionary/schemas/clinical_trial.yaml

```diff
@@ -70,277 +70,333 @@ properties:
+    type: string
 
   recruitment_status:
     description: >
-      Recruitment Status
-    type: string 
+      The recruitment status for the clinical study as a whole, based upon the status of the individual sites. If at least one facility in a multi-site clinical study has an Individual Site Status of "Recruiting," then the Overall Recruitment Status for the study must be "Recruiting."
+    enum:
+    - "Not yet recruiting"
+    - "Recruiting"
+    - "Enrolling by invitation"
+    - "Active, not recruiting"
+    - "Completed"
+    - "Suspended"
+    - "Terminated"
+    - "Withdrawn"
 
   actual_enrollment:
     description: >
-      Actual Enrollment
-    type: string 
+      The number of participants enrolled in the study
+    type: string
 
   original_estimated_enrollment:
     description: >
       Original Estimated Enrollment
     type: string
 
-  actual_study_completion_date:
+  study_start_date:
     description: >
-      Actual Study Completion Date
+      The estimated date on which the clinical study will be open for recruitment of participants, or the actual date on which the first participant was enrolled.
     type: string
 
-  actual_primary_completion_date:
+  primary_completion_date:
     description: >
-      Actual Primary Completion Date
+      The date that the final participant was examined or received an intervention for the purposes of final collection of data for the primary outcome, whether the clinical study concluded according to the pre-specified protocol or was terminated. In the case of clinical studies with more than one primary outcome measure with different completion dates, this term refers to the date on which data collection is completed for all of the primary outcomes.
     type: string
 
-  eligibility_criteria:
+  study_completion_date:
     description: >
-      Eligibility Criteria
+      The date the final participant was examined or received an intervention for purposes of final collection of data for the primary and secondary outcome measures and adverse events (for example, last participant’s last visit), whether the clinical study concluded according to the pre-specified protocol or was terminated.
     type: string
 
-  gender:
+  eligibility_criteria:
     description: >
-      Sex/Gender
+      A limited list of criteria for selection of participants in the clinical study, provided in terms of inclusion and exclusion criteria and suitable for assisting potential participants in identifying clinical studies of interest. Use a bulleted list for each criterion below the headers "Inclusion Criteria" and "Exclusion Criteria".
     type: string
 
-  ages:
+  sex:
     description: >
-      Ages: 18 Years to 99 Years
+      The sex and, if applicable, gender of the participants eligible to participate in the clinical study.
+    enum:
+    - "Male"
+    - "Female"
```

Comment:
so are we not using thesaurus definitions in this dictionary?
i mean, this definition seems more complete https://github.com/uc-cdis/covid19_datadictionary/blob/3.0.1/gdcdictionary/schemas/demographic.yaml#L64-L93

Reply:
Agreed. This does seem more complete. Do I need to add something to the terms.yaml to incorporate this or just add the block you referenced to the clinical_trials.yaml?

Reply:
looks like _terms already contains the relevant info https://github.com/uc-cdis/niaid-clinical-trials-dictionary/blob/master/gdcdictionary/schemas/_terms.yaml#L757

Reply:
Right,  but I needed to add sex. There was no external reference for this property. Only gender existed.